src/lightning_app/runners/cloud.py

```diff
@@ -136,9 +136,12 @@ def dispatch(
         if not ENABLE_PUSHING_STATE_ENDPOINT:
             v1_env_vars.append(V1EnvVar(name="ENABLE_PUSHING_STATE_ENDPOINT", value="0"))
 
-        work_reqs: List[V1Work] = []
+        works: List[V1Work] = []
         for flow in self.app.flows:
             for work in flow.works(recurse=False):
+                if not work._start_with_flow:
+                    continue
```

Comment:
Wow! This must have changed recently. I didn't know we do this by default. That is definitely weird!

Reply:
The change hasn't happened yet but will soon, this is just in preparation 😃